server/parsec/components/account.py

```diff
@@ -1,17 +1,22 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+from __future__ import annotations
 
 from enum import auto
 from uuid import uuid4
 
+from parsec._parsec import EmailValidationToken, anonymous_account_cmds
+from parsec.api import api
+from parsec.client_context import AnonymousAccountClientContext
 from parsec.types import BadOutcomeEnum
 
 
 class CreateAccountBadOutcome(BadOutcomeEnum):
     ACCOUNT_ALREADY_EXISTS = auto()
+    INVALID_EMAIL = auto()
 
 
 class BaseAccountComponent:
-    async def create(self, email: str) -> None | CreateAccountBadOutcome:
+    async def create_token(self, email: str) -> EmailValidationToken | CreateAccountBadOutcome:
```

Comment:
```suggestion
    async def create_email_valitation_token(self, email: str) -> EmailValidationToken | CreateAccountBadOutcome:
```

Better be more explicit about what this does since there is plenty of different token in this codebase ^^

Reply:
Indeed ^^

Reply:
I would also rename the `BadOutcome` for consistency:

 ```suggestion
    async def create_email_validation_token(self, email: str) -> EmailValidationToken | CreateEmailValidationTokenBadOutcome:
```